File: app/src/main/java/com/stavro_xhardha/fcbarcelonashqip/model/Standing.java
package com.stavro_xhardha.fcbarcelonashqip.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by stavro_xhardha on 21/04/2018.
 */

public class Standing {
    @SerializedName("teamName")
    private String teamName;
    @SerializedName("points")
    private int points;
    @SerializedName("goalDifference")
    private int goalDifference;
    @SerializedName("playedGames")
    private String matchesPlayed;
    @SerializedName("position")
    private int position;

    public int getPosition() {
        return position;
    }

    public void setPosition(int position) {
        this.position = position;
    }

    public String getTeamName() {
        return teamName;
    }

    public void setTeamName(String teamName) {
        this.teamName = teamName;
    }

    public int getPoints() {
        return points;
    }

    public void setPoints(int points) {
        this.points = points;
    }

    public int getGoalDifference() {
        return goalDifference;
    }

    public void setGoalDifference(int goalDifference) {
        this.goalDifference = goalDifference;
    }

    public String getMatchesPlayed() {
        return matchesPlayed;
    }

    public void setMatchesPlayed(String matchesPlayed) {
        this.matchesPlayed = matchesPlayed;
    }
}

File: app/src/main/java/com/stavro_xhardha/fcbarcelonashqip/model/Standing.kt
package com.stavro_xhardha.fcbarcelonashqip.model

import com.google.gson.annotations.SerializedName

/**
 * Created by stavro_xhardha on 21/04/2018.
 */

class Standing(@SerializedName("teamName")
               var teamName: String? = null,
               @SerializedName("points")
               var points: Int = 0,
               @SerializedName("goalDifference")
               var goalDifference: Int = 0,
               @SerializedName("playedGames")
               var matchesPlayed: String? = null,
               @SerializedName("position")
               var position: Int = 0)
